airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -71,6 +71,9 @@ def __init__(
         aws_conn_id: str = "aws_default",
         steps: list[dict] | str | None = None,
         wait_for_completion: bool = False,
+        cancel_existing_steps: bool = True,
```

Comment:
Set default for `cancel_existing_steps` to True might broke someone pipeline.
```suggestion
        cancel_existing_steps: bool = False,
```

Reply:
Handled, thanks 
Now it makes all the added parameters considered to be optional and wouldnot break existing pipelines